src/main/java/com/segment/analytics/android/integrations/google/analytics/GoogleAnalyticsIntegration.java

```diff
@@ -180,31 +185,41 @@
     sendProductEvent(event, category, properties);
 
     if (COMPLETED_ORDER_PATTERN.matcher(event).matches()) {
+
+      // initialize instance of ScreenViewBuilder class
+      ScreenViewHitBuilder builder = new ScreenViewHitBuilder();
```

Comment:
I see, I do think an event would be more appropriate here. Otherwise I'm worried this would inflate screen counts for users (since previously it wouldn't create a screen hit).